src/ui/ui_settings/ui_settings_mixin.js

```diff
@@ -0,0 +1,17 @@
+import { UiSettings } from './ui_settings';
+import { mirrorStatus } from './mirror_status';
+
+export function uiSettingsMixin(kbnServer, server, config) {
```

Comment:
The mixin pattern is old as time (4.2) and the idea was that every mixin was starting with `const config = server.config()` so we just passed it in as an arg 🤷‍♂️